Tools/Scripts/build-and-collect-pgo-profiles

```diff
@@ -28,18 +48,61 @@ mkdir -p "$BASE/motionmark"
 mkdir -p "$BASE/output"
 mkdir -p "$BASE/Internal/WebKit/WebKitAdditions/Profiling/"
 
-cd ../Internal
+cd Internal
```

Comment:
Where are we expecting this script to be run?

Reply:
A directory contains Internal and OpenSource I think.